src/main/resources/templates/email/user/addedToOrganization/body.ftlh.mjml

```diff
@@ -23,7 +23,7 @@
                     ${strings("notification.user.addedToOrganization.email.body.2", organization.name)}
                 </mj-text>
                 <mj-text mj-class="text-body03" padding-top="32px">
-                ${strings("notification.user.addedToOrganization.email.body.3", admin.fullName, admin.email)}</mj-text>
+                ${strings("notification.user.addedToOrganization.email.body.3", admin.fullName)}</mj-text>
```

Comment:
@sgrimm I need to show the user name here, not the admin name. Do you know if it is available here?

Reply:
Not at the moment, but it's easy to add to the model. I'll do that in a separate PR right now. Up to you whether you want to merge this and then do a followup, or wait for my PR and rebase this one.

Reply:
Thanks, I will merge this one and create a new one after your PR is merged